src/credentials/CHIPCert.cpp

```diff
@@ -1446,5 +1427,33 @@ CHIP_ERROR CertificateValidityPolicy::ApplyDefaultPolicy(const ChipCertificateDa
     }
 }
 
+CHIP_ERROR ValidateChipNI(const ByteSpan & ni)
```

Comment:
Please let's not use "Chip" instead of "Matter" for any new methods.

Reply:
My personal vote is keep it consistent. If in future we move chip to matter, then we do that once, and rip the bandaid off. For now, everything is chip, and I suggest keeping it that way.

Reply:
I will call this `ValidateNetworkIdentity` for now, probably doesn't need either CHIP or Matter in the name.

Reply:
On second thoughts, the `Chip` in the name here and other methods like `ValidateChipRCAC`, `ConvertChipCertToX509Cert` refers to the data format (Chip TLV vs X.509 DER), so I should probably keep it.